scripts/imgtool/keys.py

```diff
@@ -0,0 +1,129 @@
+"""
+Cryptographic key management for imgtool.
+"""
+
+from __future__ import print_function
```

Comment:
Yeah, it was an artifact of me writing keys.py before starting the rest of the code, or thinking about it.

Is there a way to bomb quickly if the script is run in python2?